src/target/unix.rs

```diff
@@ -15,6 +16,10 @@ pub trait UnixTarget {
         ptrace::cont(self.pid(), None)?;
         Ok(())
     }
+
+    /// Returns the current snapshot view of this debugee process threads
+    fn threads(&self)
```

Comment:
Why add threads function declaration to UnixTarget if no code is shared between unix targets? Shouldn't `fn threads()` be specific to Linux target just like read()?

Reply:
I imaged all UnixTarget would have threads, and that you might possibly want it there... lol. 
I'll stop imagining things :smile:  ... Removed the code and added it to the LinuxTarget.